Cargo.lock

```diff
@@ -9338,8 +9339,7 @@ dependencies = [
 [[package]]
 name = "substrate-wasm-builder"
 version = "3.0.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "79091baab813855ddf65b191de9fe53e656b6b67c1e9bd23fdcbff8788164684"
+source = "git+https://github.com/paritytech/substrate#39278096356942a087213b0f21c976ac100cb8f8"
 dependencies = [
```

Comment:
These changes don't seem relevant to this PR -- is this what you expected, @crystalin ?